### ISSUE TITLE
Occurrence to session link wrong

### ISSUE THREAD

**Issue by LDalby** [RolnickLab/antenna#443]
From the occurrence view, click on an occurrence, then click the crop to jump to session view. 
The highlighted crop area is not the correct occurrence.

<img width="818" alt="image" src="https://github.com/RolnickLab/ami-platform/assets/2366579/bd18158c-ebfb-470b-9586-d40943b7496b">

<img width="818" alt="image" src="https://github.com/RolnickLab/ami-platform/assets/2366579/7ceb1478-2361-4f1d-83b1-357238ed18f5">

**Comment by annavik** [comment]
Hi @LDalby ! I had some troubles reproducing this, is it still a problem? I tested this one (think same as in your example):
https://app.preview.insectai.org/projects/16/occurrences/8669?ordering=-determination__name

If you don't see the detection highlighted, I think might be because the prediction score is below your selected confidence threshold. I can see this is confusing though. We plan to start show all detections, regardless of confidence threshold, see https://github.com/RolnickLab/ami-platform/issues/361

**Comment by LDalby** [comment]
Yes, to me it still looks wrong...

**Comment by annavik** [comment]
Tried to clarify :)

<img width="1102" alt="Skärmavbild 2024-07-03 kl  13 37 58" src="https://github.com/RolnickLab/ami-platform/assets/11680517/81e10c2e-1414-4277-a38e-724740232ad3">

**Comment by annavik** [comment]
But could be related to something else!

**Comment by annavik** [comment]
What does your URL say after click? https://app.preview.insectai.org/projects/16/sessions/2348?occurrence=8669&capture=4262339 or something else?

**Comment by LDalby** [comment]
Yes, but when you click the occurrence you (at least I did) expect to jump the the full image with the occurrence highlighted.
Jumping to the full image with something else highlighted is quite confusing

**Comment by LDalby** [comment]
I get this: 
https://app.preview.insectai.org/projects/16/sessions/2348?occurrence=8669&capture=4262339

**Comment by LDalby** [comment]
So I think we've confirmed that the linking of session and occurrence works as it should, but that I misunderstood the purpose of jumping to the session in the first place 🤷🏼

**Comment by annavik** [comment]
> Yes, but when you click the occurrence you (at least I did) expect to jump the the full image with the occurrence highlighted. Jumping to the full image with something else highlighted is quite confusing

Yes this is what should happen, so if something else is selected this is clearly a bug! If your confidence score slider is above 34%, I would expect nothing to be selected though (since we currently filter detections). Sorry, do you mind posting a fullscreen screenshot?

**Comment by LDalby** [comment]
![image](https://github.com/RolnickLab/ami-platform/assets/2366579/d63b1cb8-21b3-48fb-916b-c9f9c93e60ae)

**Comment by LDalby** [comment]
The mouse pointer needs to hover over the session  - but on my screen it does that unless I actively moved after the click on the occurrence

**Comment by annavik** [comment]
Thank you very much! I see why it's confusing. To summarize, I think the changes planned in https://github.com/RolnickLab/ami-platform/issues/361 will take care of this. It seems related to confidence threshold filtering out your detection.

I'll keep you posted about the updates here! :)

**Comment by annavik** [comment]
Yes, let's close this! Thanks @f-PLT
